monkey/monkey_island/cc/app.py

```diff
@@ -68,6 +71,11 @@ def init_app_config(app):
     # See https://flask.palletsprojects.com/en/1.1.x/config/#JSON_SORT_KEYS.
     app.config["JSON_SORT_KEYS"] = False
```

Comment:
Unrelated but we can get rid of this now. See #753.

Reply:
🤷 Sure!